Tasks/UseGoV1/task.json

```diff
@@ -0,0 +1,66 @@
+{
+    "id": "334727F4-9495-4F9D-A391-FC621D671474",
+    "name": "UseGo",
+    "friendlyName": "Use Go",
+    "description": "Sets up a Go environment and installs it to the PATH. Additionally provides proxy support (except for go get which is done through source control providers).",
+    "helpUrl": "https://go.microsoft.com/fwlink/?linkid=867581",
+    "helpMarkDown": "[More Information](https://go.microsoft.com/fwlink/?linkid=867581)",
+    "category": "Tool",
+    "preview": true,
+    "ecosystem": "go",
+    "runsOn": [
+        "Agent",
+        "DeploymentGroup"
+    ],
+    "author": "Microsoft Corporation",
+    "version": {
+        "Major": 1,
+        "Minor": 0,
+        "Patch": 0
+    },
+    "satisfies": [
+        "GO"
```

Comment:
Is this supposed to be uppercase? Not sure if matters/case sensitive.

Reply:
I don't think it matters, changed so it looks better